packages/notification-center/src/shared/icons/avatar-system-icons/Avatar.tsx

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable max-len */
+import React from 'react';
+
+export function Avatar(props: React.ComponentPropsWithoutRef<'svg'>) {
+  return (
+    <svg width="14" height="16" viewBox="0 0 14 16" fill="none" xmlns="http://www.w3.org/2000/svg" {...props}>
```

Comment:
It's always better to remove the `width, height` and set them in the place where the icon is used, also better to set the `fill` to `currentColor`. Please check other icons too.

Reply:
```js
width={props.width || '14'}
height={props.height || '16'}
```
Maybe something like this, so we still have default values for them? wdyt?
Also, every other icon in our design system have those fixed width/height, we would have to refactor them at some point

Reply:
yeah, let's leave it for now it's not a big thing :)